sale_product_pack/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "category": "Sales",
     "summary": "This module allows you to sale product packs",
     "website": "https://github.com/OCA/product-pack",
-    "author": "NaN·tic, ADHOC SA, Tecnativa, Odoo Community Association (OCA)",
+    "author": "NaN·tic, ADHOC SA, Tecnativa, Domatix, Odoo Community Association (OCA)",
```

Comment:
A fix does not grant authorship.
Please add yourself to the contributors list instead.

Reply:
This is pending